code/modules/food_and_drinks/food/snacks_bread.dm

```diff
@@ -234,11 +234,6 @@ GLOBAL_LIST_INIT(frying_bad_chems, list(
 		QDEL_NULL(trash)
 	. = ..()
 
-/obj/item/reagent_containers/food/snacks/deepfryholder/On_Consume(mob/living/eater)
-	if(trash)
-		QDEL_NULL(trash)
-	..()
```

Comment:
?????
This would be much better to add in a separate atomized PR.
Much easier way would be to just remove cooking oil from the list. Alternatively, you could make it so that water or some other reagent is able to wash deep-fried objects, restoring them to their original state.

Reply:
Done.